test/ufuzz.js

```diff
@@ -630,8 +630,12 @@ function _createExpression(recurmax, noComma, stmtDepth, canThrow) {
             );
             break;
           default:
+            if (rng(4) == 0) s.push('function ' + name + '(){');
+            else {
+                VAR_NAMES.push('this');
```

Comment:
Furthermore, should the `new function` create a child function, `'this'` is not necessarily a valid variable name in the child function.